.github/workflows/build.yml

```diff
@@ -117,7 +117,7 @@ jobs:
           jupyter cad --version
           # jupyterlab.browser_check will fail due to the core extension jupyterlab_pygments.
           # It does not support lab v4.0.0
```

Comment:
```suggestion
```
Should I remove these comments?

Reply:
Sounds good!

Reply:
Done in [`b639bf7`](https://github.com/jupytercad/JupyterCAD/pull/386/commits/b639bf751c1ca0ca050d22c8ca35fe8956412f61)